qa/tasks/cephfs/kernel_mount.py

```diff
@@ -339,7 +340,7 @@ def _global_inst(self):
         if self.inst is not None:
             return self.inst
 
-        client_gid = "client%d" % self.get_global_id()
+        client_gid = "client%d" % int(self.get_global_id())
```

Comment:
nit: f-string will look better here too. 

```suggestion
        client_gid = f"client{self.get_global_id()}"
```
This is will also take care of catching the type issue.[[1]](https://github.com/ceph/ceph/pull/54515/files#r1439393295)

BTW, shouldn't it be `client.123` instead of `client123`?

Reply:
There is a reason for doing it this way. See: https://github.com/ceph/ceph/pull/54515#discussion_r1439347118

Reply:
> This is will also take care of catching the type issue.[[1]](https://github.com/ceph/ceph/pull/54515/files#r1439393295)

How would that be?

Reply:
> BTW, shouldn't it be `client.123` instead of `client123`?

The semantics aren't changed with this fix. That would be a separate effort but I guess it works fine this way.

Reply:
@rishabh-d-dave ping?

Reply:
> The semantics aren't changed with this fix. That would be a separate effort but I guess it works fine this way.

Okay, cool.